FoundationsMastery/LibraryBook.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    class LibraryBook : LibraryItem
+    {
+        public string Title { get; set; }
+        public string Author { get; set; }
+        public int DatePublished { get; set; }
+        public virtual int CircPeriodDay { get; } = 14;
+    }
```

Comment:
I really like these two classes - library media are a great example of inheritance